spec/Section 4 -- Introspection.md

```diff
@@ -500,3 +510,76 @@ Fields\:
     {true}, deprecated arguments are also returned.
 - `isRepeatable` must return a Boolean that indicates if the directive may be
   used repeatedly at a single location.
+
+### The \_\_Service Type
+
+The `__Service` type is returned from the `__service` meta-field and provides
+information about the GraphQL service, most notably about its capabilities. This
+type was added after the original release of GraphQL, so older schemas may not
+support it.
```

Comment:
Do we really need the `__service` indirection vs having `__capabilities`? 
My mental model is that the endpoint is the service. It has a `__schema` and `__capabilities` are at the same level (except they can't be expressed in terms of type system).

Reply:
Yikes, I'm having an existential crisis now...

Can clients know whether `__service` is supported? As a meta field, it's not returned in introspection, right? So it's too late now, we have to retrofit everything inside `__schema`???